sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/CBDE/MLIRExporter.cs

```diff
@@ -598,6 +608,29 @@ private void ExtractBinaryExpression(ExpressionSyntax expr, ExpressionSyntax lhs
             string opName;
             switch (expr.Kind())
             {
+                case SyntaxKind.RightShiftAssignmentExpression:
+                case SyntaxKind.RightShiftExpression:
+                    opName = "shlis";
```

Comment:
I think using `opName` here is decreasing readability, since it implies you are going to use the normal pattern